software_tests

```diff
@@ -33,58 +30,37 @@ subdaily p103 2020
 
 vwc_input p033 2012
 
-make_json_input p041 0 0 0 
+make_json_input p041 0 0 0
 
 # case where station should not exist in the database
-make_json_input xxxx 0 0 0 
+make_json_input xxxx 0 0 0
 
-# case where station coordinates provided 
+# case where station coordinates provided
 make_json_input xxxx  40 -120 10  -h1 5 -h2 20 -e1 5 -e2 15
 
-#
-rm $ORBITS/2010/nav/auto0010.10n
-
```

Comment:
Simpler to not explicitly remove these files and instead assume we are running in a clean container, but this can be added if we use `rm -f` to avoid failing on nonexistent files.